update repository information as we move to Maven Central

yagee-de opened the pull request.

yagee-de committed:
update repository information as we move to Maven Central

yagee-de committed:
added EnforceManagedDepsRule to check profile

yagee-de committed:
reformatted code

yagee-de committed:
update image-tiler to 1.12

MyCoRe-Travis commented:
Something gone wrong on a777a97e967e637bc4fa089d4996c7305a1b377b! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/2605-pull_request-477

yagee-de committed:
update to selenium 3.8.1

MyCoRe-Travis commented:
Something gone wrong on 9d3bc049de607a2d1591a25ea6f83feec336af59! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/2609-pull_request-477

yagee-de committed:
update mets-model to 0.11
fixed compile error due to upgrade of image-tiler

yagee-de committed:
update to easy-sword-lib-1.04

yagee-de committed:
update to mets-validator 1.3

yagee-de committed:
update datamodel-plugin to 0.7

removed mycore artifactory
added bibsonomy repository, as bibsonomy dependencies are not on central
